env_sample.sh

```diff
@@ -0,0 +1,5 @@
+export GOOGLE_CLIENT_ID=id.apps.googleusercontent.com
+export GOOGLE_CLIENT_SECRET=secret
+export PORT=3000
+export HOST=localhost
+export URL=http://localhost:3000
```

Comment:
This is just nitpicky but Git works best with editors that put a newline at the end of the file. I'd suggest turning that feature on in your editor so that git and github don't display warnings like this one.